src/CMakeLists.txt

```
@@ -17,7 +17,7 @@ set(python_files
     pair.py
     )
 
-set(_pair_evaluators )
+set(_pair_evaluators PerturbedLennardJones)
```

Add your new evaluator to the list here. The name of the evaluator and the Python class should be the same.
```suggestion
set(_pair_evaluators PerturbedLennardJones)
```